integration-test/uport-config.js

```diff
@@ -1,2 +1,2 @@
-import { Uport, PublicPersona, Registry } from 'uport-connect'
-export { Uport, PublicPersona, Registry }
```

Comment:
@zachferland uport-js does part of that already. But the entry point will be via that.